index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<body>
+style="background-color:rgb(200,200,200);color:white">
```

Comment:
This looks like an element attribute, but what's it doing down here, all alone, on a separate line, with no corresponding HTML element?